packages/component/src/SendBox/Assets/TypingAnimation.js

```diff
@@ -1,16 +1,21 @@
+import { css } from 'glamor';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
 import ScreenReaderText from '../../ScreenReaderText';
+import useDirection from '../../hooks/useDirection';
 import useStyleSet from '../../hooks/useStyleSet';
 
 const TypingAnimation = ({ 'aria-label': ariaLabel }) => {
   const [{ typingAnimation: typingAnimationStyleSet }] = useStyleSet();
+  const [direction] = useDirection();
+  const rtlScale = direction === 'rtl' ? css({ transform: 'scale(-1)' }) : '';
```

Comment:
Cache it as `const ROOT_CSS = css({})` or something like `const RTL_SCALE_CSS`.

Also, `scale(-1, 1)` instead.